credentialAtomicQueryMTPWithRelay.go

```diff
@@ -0,0 +1,223 @@
566323371461646532839","1"],["11079657783753193230705392705354621116816727265850821275138889581344006952013","2810216968439765011805022313550651229877627560533173953238141653317463782949","1"],["9223595699905107862323593944763455270298310418726121742904582596290385433272","9520911079557483359256044915383663087282532566550161871205604799701616859307","1"],["2453637300933156151913471388005432290205475823749762795966502787713330919397","17626501447631555438471094121056146422546587207169941153578757579354519372386","1"],["20299296354547215816196515877470479939681863731126999929203505010560176743509","9281533250776611056122569213915744630645754816770048518958283772073472140960","1"],["10773220848072279657847417188973800377577406916523012303257471597902598943364","2162423021911198810139295183957485939327191234705284934267078441357391730513","1"],["11687316304710163930858174108276068508137077645171138684536577637819989177914","15802373913271345112172055673147074084248578207789928223402043152729761570662","1"],["14717865026736016145790107562972654989410323191188701625351228322129478232912","3379289964991601465224086175304724078336771028002388033931418088919646602167","1"],["3630103491436973549481486644476655795401187165578612495252654925341323447962","8698796443113080903944904244325470517170342348357668147308491158363832246881","1"],["7855908251223775013950566123433089602218003936742396081340310972297676542013","3812590038566808271947062282337716230137718665867339427359410659000480002343","1"],["18545492885942215737049004832088609634162633625911715249314853234192493684260","8936508304395979387994034905877387357848893088470013100161442079733002939122","1"],["20149428730720590202466191442107273682916377416761409639058819805842696845373","7658214767865561210921080823992466471595740913840412059889335603669688907978","1"],["17037467206547780827070608838819943193534824799961802451448725828167665594078","4053226627914797325150831941352066844415581991552704063196751278281850600387","1"],["4457027852797565531279126339365134708821461253736003228402622109684461442811","19025023227846886409179641256380543650495793721680424821244045106368807774006","1"],["16813318004569152748974305228546505264674252175673105433595486174220700447029","10654663669547620902845113369766779709965651900301120045027965542585834992979","1"],["9621536826633197929192964076463659571647817164250003769164232307600646461457","15262638283246369354482154216236494010247112459648221664943202107384057089967","1"],["3762502864025223875026796995862923691262575221773580080744695406831357254867","18338878589910975799001256302982038789810866806656365674516842659860718869148","1"],["11225180794309367441911210304921481439763985684697274090921414453445436759607","6631328651185425505979432278127892501709085972586711359675373551153501950608","1"],["16191972669592001709226032157895075073341464703091116224729793510015677586847","20499249672412562122489620652535455791259413675143117920437672873335373879284","1"]]}`
```

Comment:
change vk_alpha_1 to vk_alfa_1 . Verification library in go can't process vk_alpha_1. It's known bag